### ISSUE TITLE
[Question] How to get screen reader to read action bar items?

### ISSUE THREAD

**Issue by msamwill** [Esri/calcite-design-system#7670]
### Check existing issues

- [X] I have [checked for existing issues](https://github.com/Esri/calcite-design-system/issues) to avoid duplicates

### Summary

I can navigate to action-bar and tab through items.  The Microsoft Edge screen reader does not read the names of the items.  If I open an item it will read content inside the panel.  

### Actual Behavior

1. Open site with calcite action-bar items https://codepen.io/msamwill/pen/WNLxbbM
2. Tab to action-bar, select read aloud in browser
3. Action bar items are not read.

### Expected Behavior

Expect name of action bar item to be read out load. 

### Reproduction Sample

https://codepen.io/msamwill/pen/WNLxbbM

### Reproduction Steps

1. Open site with calcite action-bar items https://codepen.io/msamwill/pen/WNLxbbM
2. Tab to action-bar, select read aloud in browser

### Reproduction Version

1.6.1

### Relevant Info

Windows 11, Microsoft Edge, Read Aloud

### Priority impact

p4 - not time sensitive

### Calcite package

- [X] @esri/calcite-components
- [ ] @esri/calcite-components-react

### Esri team

N/A

**Comment by msamwill** [comment]
Never mind.  This appears to work fine with Narrator https://dequeuniversity.com/screenreaders/narrator-keyboard-shortcuts

**Comment by geospatialem** [comment]
This is also supported with JAWS and NVDA. It looks like Read Aloud is an Edge browser add-on, which is not supported by Calcite at this time. 

With regards to [the comment above](https://github.com/Esri/calcite-design-system/issues/7670#issuecomment-1705338563) and support with JAWS, NVDA, and Narrator, could we close the issue out @msamwill?

**Comment by geospatialem** [comment]
@msamwill With the above comments verifying support in [JAWS, NVDA](https://github.com/Esri/calcite-design-system/issues/7670#issuecomment-1706709777), and [Narrator](https://github.com/Esri/calcite-design-system/issues/7670#issuecomment-1705338563) closing out the issue. If anything else pops up let us know and we can reopen to investigate.
